### ISSUE TITLE
Check model file size after saving

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gate-lf-pytorch-json#11]
Check out example dense-pos-seq-1, creates a really huge model pickle file (31M) in comparison to the Pytorch module file (3M)

**Comment by johann-petrak** [comment]
OK, this is probably because a some variables which should get treated as transient and not get saved actually do get saved:
* the validation set should not get saved
* cuda settings should not get saved and get determined as part of loading, then should get set in the pytorch model as well
* possibly others

**Comment by johann-petrak** [comment]
This is now handled in #19
